src/blog/2023/06/node-explained-filter.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Nodes explained: Filter"
+subtitle: The filter node is incredibly versatile, let's go over some use-cases
+description: Forwarding or blocking messages based on values, change of them, or lack of change.
+date: 2023-06-15
+authors: ["zeger-jan-van-de-weg"]
+image:
+tags:
+  - posts
+  - how-to
+---
+
+The Filter node, previously called "Report by Exception", has two modes of
+operation, called deadband and narrowband.
+It is commonly used in applications where sensors constantly report data and only
+significant changes need to be communicated to the control system. Or in other
+cases the lack of change needs reporting.
```

Comment:
The last sentence is a bit awkward.  I would actually delete it or maybe reword something like

'In other cases, alerts are required if sensor data doesn't change over a period of time.'

Reply:
Reading below it doesn't seem you have an example that illustrates the use case in the last sentence?